ui/pages/confirmations/components/confirm/info/personal-sign/siwe-sign/siwe-sign.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { signatureRequestSIWE } from '../../../../../../../../test/data/confirmations/personal_sign';
+import mockState from '../../../../../../../../test/data/mock-state.json';
+import configureStore from '../../../../../../../store/store';
+
+import SIWESignInfo from './siwe-sign';
+
+const store = configureStore({
+  metamask: {
+    ...mockState.metamask,
+  },
+  confirm: {
+    currentConfirmation: signatureRequestSIWE,
+  },
+});
+
+const Story = {
```

Comment:
Did we want a story for the resources case also since it's visually different?

Reply:
I changed current story to include resources as it was hard to have 2 variations in redux state